otm-gateway.html

```diff
@@ -54,6 +54,29 @@ <h2>Status of This Document</h2>
     <section id="conformance">
     </section>
 
+    <section id="introduction" class="informative">
+      <h2>Introduction</h2>
+
+      <p>This specification describes APIs enabling digital object repository systems to manage preservation of content.
+      The interfaces described here are intended to fit within the larger context of the <a href="preservation-workflow.html">One to Many Preservation Workflow</a>.</p>
+      <p>This specification defines a set of interfaces for synchronous interactions for depositing, restoring,
+      and purging objects across multiple distributed digital preservation (DDP) systems, using a single system
+      managed by the repository administrators. Because DDPs (and the 
+      <a href="otm-bridge.html">One to Many Bridge API</a>) perform deposit and retrieval actions 
```

Comment:
The link to the Bridge spec (and the use of the full name of the Bridge) here should be duplicated in the Abstract section above (where the term is first used).